includes/admin/settings/class-wcmpbe-settings.php

```diff
@@ -99,6 +99,7 @@ class WCMPBE_Settings
     public const SETTING_CARRIER_CUTOFF_TIME              = "cutoff_time";
     public const SETTING_CARRIER_DELIVERY_DAYS_WINDOW     = "delivery_days_window";
```

Comment:
Is de `delivery_days_window` nu nog wel nodig?

Reply:
Ik denk namelijk dat deze weg kan omdat de setting is veranderd van naam.

Reply:
kijk ook even of de migratie voor wcmpbe-upgrade-migration-v4-0-0.php nog goed gaat

Reply:
Bedankt voor het wijzen op de migratie. Door de constante er nog even in te houden kun je m wel netjes meenemen in de migratie, zie de geupdatete code :-)